src/csvcubed/schema/cube-config/v1_1/schema.json

```diff
@@ -545,6 +545,13 @@
                     "enum": [
                         "observation-status"
                     ]
+                },
+                "units": {
```

Comment:
This isn't quite going to work. You'll need to add a `from_template` field on the `units` column which points to one of these values.

At that point, I think we should move to v1.2 since it is a significant change.